### PR TITLE
tests: expand R pylint rules and start re-enabling these

### PR DESCRIPTION
**Pull request by KKoukiou**

### CODE REVIEW

**Review by elkoniu — COMMENTED** [review]
LGTM for what it is worth, although my python kung-fu is not perfect :)

File: pyanaconda/argument_parsing.py [hunk 1]
```diff
@@ -307,8 +307,11 @@ def name_path_pairs(image_specs):
             name = os.path.splitext(os.path.basename(path))[0]
 
         if name in names_seen:
-            names = ("%s_%d" % (name, n) for n in itertools.count())
-            name = next(itertools.dropwhile(lambda n: n in names_seen, names))
+            for n in itertools.count():
+                candidate = f"{name}_{n}"
```
Comment (jkonecny12) [inline pyanaconda/argument_parsing.py:311]: So did we agreed to use f-strings in the codebase?

**Review by jkonecny12 — APPROVED** [review]
Looks good to me. Just one question raised below.

**Pull request merged**